### DISCUSSION TITLE
Is it possible to avoid panning the page?

### DISCUSSION THREAD

**Discussion by Xwang1976** [flxzt/rnote#511] [Q&A]
Hi to all,
I'm starting using rnote on my surface like pc running linux.
I would like it to handwrite with the included stylo, but the page is not fixed how I would like.
Is it possible to disable the page panning and zoom and just select the following pages as if it was a paper note?

**Comment by Xwang1976** [comment]
To better explain, even if I set the fixed page, it happens to involuntary change the Zoom while writing with the pen.
Is it possible to fix the size of the page, maximize it on the base of the size of the screen and  then just move to the following pages by clicking on the button to change page?

**Comment by PhilDevProg** [comment]
Disabling pinch to zoom is coming in the next version and you can already test it by building the main branch yourself. For disabling scrolling/dragging, you should open a new issue since Discussions are not the right place for specific feature requests and might get overlooked by developers.
